### DISCUSSION TITLE
Downloading more than 1 url in same webdriver.!

### DISCUSSION THREAD

**Discussion by hassanseoul123** [gildas-lormeau/SingleFile#659] [Ideas]
Hi @gildas-lormeau 
Thank you so much for this amazing utility!
I'm using SingleFile from cli.
Is it possble add a feature by which we can save multiple urls in the same webdriver by putting them in queue.
Currently we can use text file containing urls but it results in opening of multiple webdrivers at the same time.
Thanks!

**Comment by gildas-lormeau** [comment]
Hi, sorry for the late reply.

As far as I know, there's no way to handle multiple tabs reliably in Selenium. It's possible with puppeteer and playwright though. Please let me know if you have any suggestion on how to implement it.

**Comment by hassanseoul123** [comment]
Thank You For your reply!
Is it possible to loop over many URLs one by one in selenium? (in the same
tab)
